generators/client/files.js

```diff
@@ -59,6 +59,12 @@ const clientFiles = {
                     method: 'processJs',
                     renameTo: () => `${ANGULAR_DIR}core/login/login.service.ts`,
                 },
+                // profiles response returns a hyphenated key
+                {
+                    file: 'angular/src/main/webapp/app/layouts/profiles/profile.service.ts',
+                    method: 'processJs',
+                    renameTo: () => `${ANGULAR_DIR}layouts/profiles/profile.service.ts`,
```

Comment:
This is just something I noticed when deploying to heroku/using prod profile. For react there is already a custom reducer handling the hyphenated profile names. For angular there was not and therefore e.g. the database menu item was displayed.